index.html

```diff
@@ -38,6 +38,194 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
+  </p>
+  <div>
+    <ul class="list-btn">
+      <li class="css-btn">css</li>
+      <li class="html-btn">html</li>
+      <li class="boot-btn">bootsrap</li>
+      <li class="ruby-btn">ruby</li>
+    </ul>
+    <button class="btn-top" type="button">See projests</button>
+  </div>
+  <div>
+    <div  class="header-pro">
+    <h4 class="pro">Profesional Art Printing Data</h4>
+    <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+      been the industry's standard</p>
+    <ul class="pro-ul">
+      <li class="pro-html">html</li>
+      <li class="pro-boot">bootsrap</li>
+      <li class="pro-rubby">Ruby</li>
+    </ul>
+  </div>
+    <button class="pro-btn" type="button">See Projests</button>
+    <div  class="header-pro">
+      <h4 class="pro">Profesional Art Printing Data</h4>
+      <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+        been the industry's standard</p>
+      <ul class="pro-ul">
+        <li class="pro-html">html</li>
+        <li class="pro-boot">bootsrap</li>
+        <li class="pro-rubby">Ruby</li>
+      </ul>
+    </div>
+      <button class="pro-btn" type="button">See Projests</button>
+      </div>
+      <div>
+        <div  class="header-pro">
+          <h4 class="pro">Profesional Art Printing Data</h4>
+          <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+          <ul class="pro-ul">
+            <li class="pro-html">html</li>
+            <li class="pro-boot">bootsrap</li>
+            <li class="pro-rubby">Ruby</li>
+          </ul>
+        </div>
+        <button class="pro-btn" type="button">See Projests</button> 
+      </div>   
+      <div>
+        <div  class="header-pro">
+          <h4 class="pro">Profesional Art Printing Data</h4>
+          <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+          <ul class="pro-ul">
+            <li class="pro-html">html</li>
+            <li class="pro-boot">bootsrap</li>
+            <li class="pro-rubby">Ruby</li>
+          </ul>
+        </div>
+        <button class="pro-btn" type="button">See Projests</button> 
+      </div>  
+      <div>
+        <div  class="header-pro">
+          <h4 class="pro">Profesional Art Printing Data</h4>
+          <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+          <ul class="pro-ul">
+            <li class="pro-html">html</li>
+            <li class="pro-boot">bootsrap</li>
+            <li class="pro-rubby">Ruby</li>
+          </ul>
+        </div>
+        <button class="pro-btn" type="button">See Projests</button> 
+      </div>  
```

Comment:
- [ ] **[OPTIONAL]** There are three ✌🏽 more cards than in the Figma template, please delete them to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

![image](https://user-images.githubusercontent.com/58642949/185726743-002946c7-8f48-450e-b72a-74ef917a5bb5.png)